src/api/app/models/group.rb

```diff
@@ -122,16 +122,7 @@ def involved_projects
 
   # lists packages maintained by this user and are not in maintained projects
   def involved_packages
-    # just for maintainer for now.
-    role = maintainer_roler
-
-    projects = involved_projects_ids
-    projects << -1 if projects.empty?
-
-    # all packages where group is maintainer
-    packages = Relationship.where(group_id: id, role_id: role.id).joins(:package).where.not('packages.project_id' => projects).pluck(:package_id)
-
-    Package.where(id: packages).where.not(project_id: projects)
```

Comment:
@danidoni hmm, the `where.not(project_id: projects)` is gone now, maybe that is responsible for the duplicates?

Reply:
Let me check that

Reply:
What exactly is duplicated?

Reply:
On the group requests page we get multiple request rows, because we ask for requests having request actions or not and having reviews or not, and the query is returning one request row for each request action a request may have.